src/web/core/src/main/java/org/geoserver/web/publish/PublishedConfigurationPage.java

```diff
@@ -1,444 +1,446 @@
-     */
-    public void updatePublishedInfo(T info) {
-        myModel.setObject(info);
-    }
-
-    protected abstract class ListEditTabPanel extends PublishedEditTabPanel<T> {
-
-        private static final long serialVersionUID = -7279044666531992361L;
-
-        public ListEditTabPanel(String id) {
-            super(id, myModel);
-
-            ListView<?> list = createList("theList");
-
-            // do this or die on validation (the form element contents will
-            // reset, the edit will be lost)
-            list.setReuseItems(true);
-            add(list);
-        }
-
-        protected abstract ListView<?> createList(String id);
-    }
-
-    protected class PublishingEditTabPanel extends ListEditTabPanel {
-        private static final long serialVersionUID = -6575960326680386479L;
-
-        public PublishingEditTabPanel(String id) {
-            super(id);
-        }
-
-        @Override
-        public ListView<PublishedConfigurationPanelInfo<T>> createList(String id) {
-            List<PublishedConfigurationPanelInfo<T>> pubPanels =
-                    filterPublishedPanels(
-                            getGeoServerApplication()
-                                    .getBeansOfType(PublishedConfigurationPanelInfo.class));
-            ListView<PublishedConfigurationPanelInfo<T>> pubPanelList =
-                    new ListView<PublishedConfigurationPanelInfo<T>>(id, pubPanels) {
-                        private static final long serialVersionUID = 1L;
-
-                        @Override
-                        protected void populateItem(
-                                ListItem<PublishedConfigurationPanelInfo<T>> item) {
-                            PublishedConfigurationPanelInfo<T> panelInfo =
-                                    (PublishedConfigurationPanelInfo<T>) item.getModelObject();
-                            try {
-                                PublishedConfigurationPanel<T> panel =
-                                        panelInfo
-                                                .getComponentClass()
-                                                .getConstructor(String.class, IModel.class)
-                                                .newInstance("content", myModel);
-                                item.add((Component) panel);
-                            } catch (Exception e) {
-                                throw new WicketRuntimeException(
-                                        "Failed to add pluggable layer configuration panels", e);
-                            }
-                        }
-                    };
-            return pubPanelList;
-        }
-    }
-
-    @Override
-    protected ComponentAuthorizer getPageAuthorizer() {
-        return ComponentAuthorizer.WORKSPACE_ADMIN;
-    }
-
-    public void setInputEnabled(boolean inputEnabled) {
-        this.inputEnabled = inputEnabled;
-        get("publishedinfo:save").setVisible(inputEnabled);
-    }
-}

```

Comment:
This PR still shows every line changed, please fix line feeds. I have git client configuration instructions in the [developers guide](https://docs.geoserver.org/latest/en/developer/source.html#git-client-configuration) recently so please double check.